db-engine/pom.xml

```diff
@@ -275,6 +275,51 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>add-platform-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>${spark.major.version}/src/main/java</source>
```

Comment:
If db-engine has to depend on Spark there is something seriously wrong in the design.

Reply:
${spark.major.version} is not defined when only core modules are built (_mvn -Pcore_).
The _spliceengine-core_ build in Jenkins will fail.